### ISSUE TITLE
Redis 6.X : Required ACLs

### ISSUE THREAD

**Issue by mmorel-35** [redisson/redisson#3517]
Hello !
What are the minimal required ACLs tu use redisson and the org.redisson.spring.cache.RedissonCache in particular.
I haven't found any configuration about this.
Moreover, I have activated "useScriptCache" option.

Thank you

**Comment by mrniko** [comment]
It uses 4 objects:

org.redisson.RedissonMapCache#getTimeoutSetName()
org.redisson.RedissonMapCache#getLastAccessTimeSetName()
org.redisson.RedissonMapCache#getIdleSetName()
org.redisson.RedissonMapCache#getOptionsName()

You need to grant access for all of them.

**Comment by mmorel-35** [comment]
Let me rephrase my question. In redis you have [acls](https://redis.io/topics/acl#acl-rules).
So what i need to do is to define a user like this
ACL SETUSER redisson-user on ...

What i'm wondering is what are the required redis commands and the required keys or key pattern that i need to allow redisson-user to fonction.

The point being that several applications with different right are going to access redis but those application don't need every commands and every keys to work. And we don't want that an application modify a cache that it's not allowed to access.

**Comment by mrniko** [comment]
RedissonMapCache uses almost all hash and zset commands. As for keys, there is no single prefix for RMapCache object used by RedissonCache. It should be improved in future releases.

**Comment by mmorel-35** [comment]
If I try to summarise : 
concerning the keys : 
<key_name>  => type : HASH
{<key_name>}:redisson_options => type : HASH
redisson__idle__set:{<key_name>} => type : ZSET
redisson__map_cache__last_access__set:{<key_name>} => type : ZSET
redisson__timeout__set:{<key_name>} => type : ZSET
redisson__execute_task_once_latch:{<key_name>} => type : STRING

which gives us : 
```
+@hash ~<key_name> {<key_name>}:redisson_options | +@set ~redisson__idle__set:{<key_name>} ~redisson__map_cache__last_access__set:{<key_name>} ~redisson__timeout__set:{<key_name>} | +@string ~redisson__execute_task_once_latch:{<key_name>}
```

What do you think ?

**Comment by mrniko** [comment]
pattern seems correct. I'll extend API for this feature.
